src/seedu/addressbook/ui/Formatter.java

```diff
@@ -63,4 +64,15 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public static String getPrintableString(Printable... printables){
+        String result = "";
+        for (Printable p : printables) {
+            result += p.getPrintableString() + ", ";
```

Comment:
Java has a string builder that could be used here, `StringJoiner` class. You can consider using it in the future, makes string manipulation easier.